### ISSUE TITLE
Cookie not available in FastLoginController.php

### ISSUE THREAD

**Issue by captenmasin** [m1guelpf/laravel-fastlogin#14]
I'm creating a new issue here because I butchered the last one 

Registering a key/credential is fine and I can see the `X-FastLogin` cookie exists when viewing in `inspect element` however when trying to **login** with the auth details, I get a 500 error. I dug around a little and it seems (for whatever reason) the `X-FastLogin` cookie is not available and I get this error
```
message: "Webauthn\\PublicKeyCredentialDescriptor::__construct(): Argument #2 ($id) must be of type string, null given, called in /{site directory}/vendor/m1guelpf/laravel-fastlogin/src/Http/Controllers/FastLoginController.php on line 78"
```

I tried DD-ing the cookie in `FastLoginController.php` and the cookie just seems to be null

<img width="884" alt="Screenshot 2021-07-05 at 1 45 45 pm" src="https://user-images.githubusercontent.com/25454872/124474012-ea8f6b80-dd97-11eb-9401-34889eef85a5.png">

<img width="1391" alt="Screenshot 2021-07-05 at 1 46 01 pm" src="https://user-images.githubusercontent.com/25454872/124473996-e6634e00-dd97-11eb-9f07-5379e3fa6541.png">

<img width="703" alt="Screenshot 2021-07-05 at 1 46 15 pm" src="https://user-images.githubusercontent.com/25454872/124474017-ebc09880-dd97-11eb-81da-14c73da50514.png">

**Comment by pankajjha** [comment]
Same issue with me I am using Laravel Jetstream + Inertia + Vue

**Comment by captenmasin** [comment]
@pankajjha yes I'm also using Inertia + Vue, I wonder if this will have anything to do with it

**Comment by captenmasin** [comment]
@pankajjha I've just resolved this! It seems adding 
```
    protected $except = [
        'X-FastLogin'
    ];
```

To your `EncryptCookies` middleware fixes it

**Comment by captenmasin** [comment]
@m1guelpf would it be worth adding that information to the README do you think?

**Comment by m1guelpf** [comment]
@captenmasin Would be open to a PR that excludes the cookie from encryption using [the `disableFor` method](https://github.com/laravel/framework/blob/8.x/src/Illuminate/Cookie/Middleware/EncryptCookies.php#L53)

**Comment by captenmasin** [comment]
@m1guelpf excellent, I'll jump on that

**Comment by m1guelpf** [comment]
Fixed in #15
